src/extension.ts

```diff
@@ -13,8 +13,10 @@ export function activate(context: vscode.ExtensionContext) {
 		clearDecorations();
 	});
 
+	const output :vscode.OutputChannel = vscode.window.createOutputChannel("Austin");
```

Comment:
Can we create this inside the `AustinController` so that we don't have to pass it in via the constructor?